### PR TITLE
[redis-ha]: remove memory/cpu limits for redis

### PR DESCRIPTION
**Pull request by ssalaues**
**What does this PR do, and why do we need it?**
Removes hard memory limit on redis

**Which issue does this PR fix?**

fixes ZENKO-1189

**Special notes for your reviewers**:

### CODE REVIEW

**Review by anurag4DSB — APPROVED** [review]

File: kubernetes/zenko/resources.yaml [hunk 1]
```diff
@@ -64,13 +64,12 @@ prometheus:
    resources: {}
 
 redis-ha:
-  server:
```
Comment (rahulreddy) [inline kubernetes/zenko/resources.yaml:67]: Why is this changing from server => redis?

**Review by JianqinWang — APPROVED** [review]

File: kubernetes/zenko/resources.yaml (hunk 1 above)
Comment (ssalaues) [reply kubernetes/zenko/resources.yaml:67]: The file was out of date. `server` was how the old redis chart organized these values

File: kubernetes/zenko/values.yaml [hunk 2]
```diff
@@ -164,6 +164,8 @@ redis-ha:
   replicas: *nodeCount
   redis:
     masterGroupName: zenko
+    resources:
+      limits: {}
```
Comment (giacope) [inline kubernetes/zenko/values.yaml]: This doesn't work. Try `""` on the children level.

**Review by giacope — CHANGES_REQUESTED** [review]

File: kubernetes/zenko/values.yaml (hunk 2 above)
Comment (ssalaues) [reply kubernetes/zenko/values.yaml]: Fixed with `null`

**Review by giacope — APPROVED** [review]

**Pull request merged**